functional-tests/devscripts/run_tests.sh

```diff
@@ -78,6 +88,9 @@ else
     RESULT=$?
 fi
 
+JOB_TYPE="periodic-tests"
+archiveArtifacts
```

Comment:
What about moving archiving artifacts at the end of cico_rhche_prcheck.sh? It would make sense from a flow view - firstly tests take place, secondly artifacts are sent. And it wouldn't be needed to source ```prepare_env_utils.sh``` here.